### DISCUSSION TITLE
PDF文件翻译后无法下载，这是一大败笔

### DISCUSSION THREAD

**Discussion by crazy-lion** [immersive-translate/immersive-translate#802] [Ideas]
### 新功能 | New Feature

1.翻译后可以下载翻译好的PDF文件
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.因为有太多外语PDF文件了，这个功能非常有用
2.
3.

**Comment by theowenyoung** [comment]
最新版本是可以的，你是哪个版本呢？

**Comment by crazy-lion** [comment]
更新后可以了，很不错！谢谢&nbsp;


861099255
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月3日(星期四) 凌晨2:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [immersive-translate/immersive-translate] PDF文件翻译后无法下载，这是一大败笔 (Discussion #802)





 
最新版本是可以的，你是哪个版本呢？
